crm_rma_stock_location/views/stock_picking.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+        <!-- INHERITED VIEW FOR THE OBJECT : stock_picking -->
+        <record id="picking_in_form" model="ir.ui.view">
+            <field name="name">crm_claim_rma.picking_in_form</field>
+            <field name="model">stock.picking</field>
+            <field name="inherit_id" ref="stock.view_picking_form"/>
+            <field name="arch" type="xml">
+                <xpath expr="//field[@name='state']" position="before">
+                        <button name="%(action_stock_picking_from_claim_picking)d"
```

Comment:
@pedrobaeza but both options give the same result, which is the difference? I personally prefer an action per view (as it is). Which is the added value of change that dude?